examples/applications/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# CMakeLists.txt
+
+cmake_minimum_required(VERSION 3.26)
+project(RISCVTests)
+
+set(COMPILE_FLAGS "-fsycl -Wall ${WIN_FLAG}")
+set(LINK_FLAGS "-fsycl")
+
+function(run_riscv_example_with_dpcpp)
+# To compile in a single command:
```

Comment:
Will we test this in some way or as that part of ock_demo?

Reply:
In ock demo. I have already written step, but I changed quite a few things in the networks PR. so was waiting for that to get in to add these tests. I tested them in the normal ock build here ->  https://github.com/codeplaysoftware/oneapi-construction-kit/actions/runs/7570258362/job/20615256925